### DISCUSSION TITLE
Welcome to SRAMS-frontend Discussions!

### DISCUSSION THREAD

**Discussion by TobiasABay** [P3-Hype/SRAMS-frontend#49] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MathiasFM** [comment]
Very good Tobias :)

**Reply by TobiasABay** [reply]
Thanks 💋💋💋

**Comment by alexrefshauge** [comment]
Very very good :)

**Comment by TobiasABay** [comment]
Thanks @alexrefshauge
